resin/settings.py

```diff
@@ -35,9 +39,9 @@ class Settings(object):
         'api_version': 'v3',
         'data_directory': Path.join(HOME_DIRECTORY, '.resin'),
         # cache time : 1 week in milliseconds
-        'image_cache_time': (1 * 1000 * 60 * 60 * 24 * 7),
+        'image_cache_time': str(1 * 1000 * 60 * 60 * 24 * 7),
         # token refresh interval: 1 hour in milliseconds
-        'token_refresh_interval': (1 * 1000 * 60 * 60)
+        'token_refresh_interval': str(1 * 1000 * 60 * 60)
```

Comment:
Why are these strings? It looks like everywhere they're used, we really want integers.

I'm guessing it's because when we read them from a config file, they'd end up as strings anyway, and consistency is good (totally agree), but if that's the case, we should briefly comment this above ('all config values must be strings because...), or we're definitely going to forget that and add non-string config items in here in future.

Reply:
@pimterry makes sense, I've added a comment about this.